src/test/java/org/carlspring/cloud/storage/s3fs/fileSystemProvider/SetAttributeTest.java

```diff
@@ -38,24 +38,4 @@ void readAttributesObject()
         assertNotNull(exception);
     }
 
-    /**
-     * create a new file system for s3 scheme with fake credentials
-     * and global endpoint
-     *
-     * @return FileSystem
-     * @throws IOException
-     */
-    private S3FileSystem createNewS3FileSystem()
```

Comment:
:+1: for removing this dead code, but you forgot to remove the no longer used imports:

```
import org.carlspring.cloud.storage.s3fs.S3FileSystem;
import org.carlspring.cloud.storage.s3fs.S3UnitTestBase;
import org.carlspring.cloud.storage.s3fs.util.S3EndpointConstant;

import java.io.IOException;
import java.nio.file.FileSystemNotFoundException;
import java.nio.file.FileSystems;
```

should become

```
import org.carlspring.cloud.storage.s3fs.S3UnitTestBase;
import org.carlspring.cloud.storage.s3fs.util.S3EndpointConstant;

import java.io.IOException;
```